### ISSUE TITLE
Process leak of ssh processes in yugaware

### ISSUE THREAD

**Issue by tylarb** [yugabyte/yugabyte-db#5569]
Version 2.2.2.0

After installing yugaware and leaving it up for a few days, maybe a couple of weeks, it appears there is a process leak for SSH, creating hundreds of Z state processes

```
~/tpcc$ ps aux | awk '$8 ~/Z/ {print $0}'  | wc -l
176
~/tpcc$ ps aux | awk '$8 ~/Z/ {print $0}' | head
root      2434  0.0  0.0      0     0 ?        Zs   Aug31   0:00 [ssh] <defunct>
root      2435  0.0  0.0      0     0 ?        Z    Aug31   0:00 [ssh] <defunct>
root      2436  0.0  0.0      0     0 ?        Zs   Aug31   0:00 [ssh] <defunct>
root      2437  0.0  0.0      0     0 ?        Z    Aug31   0:00 [ssh] <defunct>
root      2439  0.0  0.0      0     0 ?        Zs   Aug31   0:00 [ssh] <defunct>
root      2440  0.0  0.0      0     0 ?        Z    Aug31   0:00 [ssh] <defunct>
root      2441  0.0  0.0      0     0 ?        Zs   Aug31   0:00 [ssh] <defunct>
root      2442  0.0  0.0      0     0 ?        Z    Aug31   0:00 [ssh] <defunct>
root      2444  0.0  0.0      0     0 ?        Zs   Aug31   0:00 [ssh] <defunct>
root      2445  0.0  0.0      0     0 ?        Z    Aug31   0:00 [ssh] <defunct>
```

`ps axjf` output shows the same PPID for every process:

```
31391 10119 10119 10119 ?           -1 Zs       0   0:00  |       \_ [ssh] <defunct>
31391 10120 10119 10119 ?           -1 Z        0   0:00  |       \_ [ssh] <defunct>
31391 10121 10121 10121 ?           -1 Zs       0   0:00  |       \_ [ssh] <defunct>
31391 10122 10121 10121 ?           -1 Z        0   0:00  |       \_ [ssh] <defunct>
31391 10178 10178 10178 ?           -1 Zs       0   0:00  |       \_ [ssh] <defunct>
31391 10179 10178 10178 ?           -1 Z        0   0:00  |       \_ [ssh] <defunct>
```

Which is:
```
$ ps ax | grep 31391  | cut -c -150
 1553 pts/7    S+     0:00 grep --color=auto 31391
31391 ?        Ssl   40:20 /usr/lib/jvm/java-1.8-openjdk/jre/bin/java -XX:PermSize=1024m -XX:MaxPermSize=1024m -Duser.dir=/opt/yugabyte/yugaware -Dcon
$ ps ax | grep 31391  | rev | cut -c -150 | rev
 2066 pts/7    S+     0:00 grep --color=auto 31391
yte/yugaware/lib/org.slf4j.jul-to-slf4j-1.7.25.jar:/opt/yugabyte/yugaware/lib/yugaware.yugaware-2.2.2.0-b0-assets.jar play.core.server.ProdServerStart
```

So I'm expecting that somewhere inside of the yugaware java, a process "start" is called without calling a process "wait" , or instead of using some run syntax which calls start and wait. 

I'm not familiar with the java syntax, but in golang:

https://golang.org/pkg/os/exec/#Cmd.Run

If you call Cmd.Start without a Cmd.Wait, once the process is finished, it will enter Z state until the Parent PID is killed, and the dead processes are then reaped by the OS.

This will eventually cause a user to be unable to create new processes, which may prevent them from even logging into the system (creating a new login process), once pid limit in ulimit is hit.

**Comment by kmuthukk** [comment]
cc: @streddy-yb .

**Comment by tylarb** [comment]
Believe this is a duplicate of https://github.com/yugabyte/yugabyte-db/issues/8165
